core/src/main/java/io/questdb/std/str/Path.java

```diff
@@ -333,6 +333,25 @@ public Path trimTo(int len) {
         return this;
     }
 
+    public Path parent() {
```

Comment:
Can you remove this method all together? 
We usually have integer to trip the length to instead of doing character scans to find the parent folder, your usage is fine but it would be better to avoid it.

Reply:
adding this method makes the code so much readable and for this use case  (not frequent, IO bound, user triggered) I think readability trumps the nanoseconds we would gain.